python/py-semver/Portfile

```diff
@@ -17,23 +17,24 @@ long_description    ${description}
 
 homepage            https://github.com/k-bx/python-semver
 
-checksums           sha256  723be40c74b6468861e0e3dbb80a41fc3b171a2a45bf956c245304773dc06055 \
-                    rmd160  d376c882ee915ed254152bcc732fe2f91f42fa49 \
-                    size    34508
+checksums           sha256  b08a84f604ef579e474ce448672a05c8d50d1ee0b24cee9fb58a12b260e4d0dc \
+                    rmd160  8e2e82dff987cc8d97e2769e44546d8d009148b0 \
+                    size    21661
 
 python.versions     37 38
 
 if {${name} ne ${subport}} {
-    depends_build-append \
+    depends_lib-append \
                     port:py${python.version}-setuptools
```

Comment:
Why this change ? Setuptools is commonly a build dep. only. Does this port really need it at runtime too ?

Reply:
Starting from 2.9.1 semver introduces a console script and hence setuptools is required at runtime.